### DISCUSSION TITLE
Spare PCB's available (all gone)

### DISCUSSION THREAD

**Discussion by amsmith525** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#404] [General]
I have 4 spare PCB's from my JLCPCB order - free, you pay shipping. Will ship anywhere.
![spare pcb](https://user-images.githubusercontent.com/8604026/200994186-6d7fda83-9e95-4777-8577-830f9e75aa35.png)

**Comment by TazmanianV** [comment]
Hi @amsmith525 

Wat will it cost to send 2 PCB's to Belgium?
Thx!

**Reply by amsmith525** [reply]
~$18 usd

**Reply by TazmanianV** [reply]
To where may I transfer the money?
Can I email you private for my address details?

**Reply by amsmith525** [reply]
https://paypal.me/amsmith525
![image](https://user-images.githubusercontent.com/8604026/202931509-07832a91-4595-4efc-9045-c9537d73e2f1.png)

Please put "Gift PCB" followed by your address details in the 'What's this payment for?' section.

**Reply by TazmanianV** [reply]
I just did the payment :)

**Reply by amsmith525** [reply]
Shipment details sent in paypal message!

**Reply by TazmanianV** [reply]
Hi Amsmith,

Unfortunately, I have not yet received the package.
Can you contact USPS?
I am not getting any info from them because I am not the sender of the package.
Via paypal I had also sent you a message.

**Reply by amsmith525** [reply]
Hey - the tracking just updated today. It is still being held in brussels customs. I apologize for the delay -- I am still holding the 2 PCB's in case they do not make it through. I will update you as soon as I have more information!

**Reply by amsmith525** [reply]
Held in Customs

BRUSSELS EUROPEAN MAIL CENTRE, BELGIUM

December 28, 2022, 12:27 pm

**Reply by TazmanianV** [reply]
Hi amsmith525,

I received a letter from customs today. 
The parcel will not be sent further until I can provide an invoice on which the import fee is calculated.
Or a proof that it is a gift then an exemption from import fees can be requested.
This must be done within 14 days otherwise the package will be sent back.
Can you provide this to me?

**Reply by amsmith525** [reply]
Sure thing - is there a form I need to fill out?

**Reply by amsmith525** [reply]
I have created a PDF with the customs declaration and invoice - please let me know what email to send to as well as if you need a specific form completed

**Reply by TazmanianV** [reply]
No there is no form.
The best thing is that you should send me an email saying what the package contains, what its value is and that it is a gift. 
See info from customs:

_"My parcel is a gift, now what?

A shipment from private person to private person with a value of up to 45 euros can be exempted from import charges if it is a gift. Provided that it is not one of the following products: alcoholic products, perfume, toilet water, tobacco and tobacco products. However, the sender must confirm this. Instead of an invoice, include a screenshot (from a conversation) or email in which the sender states what is in the shipment, its value and that it is a gift."_
 
You may send it to verstraetenmichael@hotmail.com

Thanks

**Reply by TazmanianV** [reply]
> I have created a PDF with the customs declaration and invoice - please let me know what email to send to as well as if you need a specific form completed

Ok thats even beter. 
Verstraetenmichael@hotmail.com

**Reply by TazmanianV** [reply]
It took a while but finally received this week the pcbs ;)

Thanks!

**Reply by amsmith525** [reply]
Great! I'll mark the remaining PCB's as available now :)

**Comment by amsmith525** [comment]
2 PCB's left

**Reply by amsmith525** [reply]
----These are being held until I confirm my last shipment gets through----

**Reply by amsmith525** [reply]
The remaining 2 PCB's are now available once more!

**Comment by tbwalkeriii** [comment]
Still have 2 left?  I'm in Panama City, Florida.  How much do you want for them shipped to me?

**Reply by amsmith525** [reply]
~$12

**Reply by amsmith525** [reply]
----These are being held until I confirm my last shipment gets through----

**Reply by amsmith525** [reply]
The remaining 2 PCB's are now available once more!

**Comment by lux-k** [comment]
Still available? How much to ship 2 boards to Blackwood, NJ 08012?

**Reply by amsmith525** [reply]
Approximately $12

**Reply by lux-k** [reply]
Sounds awesome. I can Paypal you $14 (to make sure it's all covered) using your above Paypal link, if that's OK.

**Reply by amsmith525** [reply]
Sure thing - send address details

**Reply by lux-k** [reply]
In your Paypal inbox. Thank you.

**Reply by amsmith525** [reply]
USPS Priority Mail Tracking # 9405509205568561159605

**Reply by lux-k** [reply]
Great. Thanks again.

**Reply by lux-k** [reply]
Thanks. Arrived today. Soldered about 15m ago and all works. Much appreciated.

**Reply by amsmith525** [reply]
A+

Glad it showed up quick!

**Comment by amsmith525** [comment]
All PCB's have been shipped! Thanks!
